### ISSUE TITLE
Support Firefox…  (Not trolling, honest!  😉)

### ISSUE THREAD

**Issue by bwinton** [jed/osx-browser-vm#1]
It seems like it should be possible, using @harthur's [firefox-client](https://github.com/harthur/firefox-client) library, and the DevTools remote API.  Not as easy as adding an extra target to the AppleScript call, but hopefully not _too_ hard…  🙂

**Comment by bwinton** [comment]
(Also, feel free to close this if you don't think it would be worth the effort.)

**Comment by jed** [comment]
that looks like a very different beast. i'll leave this open in case anyone wants to tackle it, but it'd be an entirely different implementation.

**Comment by jed** [comment]
(would probably be more productive for [Firefox to support scripting](https://bugzilla.mozilla.org/show_bug.cgi?id=5704))

**Comment by bwinton** [comment]
Yup, but since that bug is old enough to drink (in Quebec), I'm not going to hold my breath waiting for it to get fixed…  :wink:
